packages/paste-core/components/combobox/src/extractPropsFromState.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {useComboboxPrimitive} from '@twilio-paste/combobox-primitive';
+import type {UseComboboxPrimitiveStateChange, UseComboboxPrimitiveReturnValue} from '@twilio-paste/combobox-primitive';
+import {isEmpty} from 'lodash';
```

Comment:
Be careful here. For component packages we tend to only use the individual lodash packages, otherwise you end up compiling all of lodash.

This is why the package size has blown up I suspect.

![image](https://user-images.githubusercontent.com/368249/152909719-ecfaed6d-6e06-4444-a5ea-37a60c9e986e.png)

Reply:
Yea this was an auto import thing. Thanks, I'll change to `import isEmpty from 'lodash/isEmpty';` 👍